src/node/internal/internal_inspect.ts

```diff
@@ -661,7 +665,7 @@ inspect.colors = {
   bgMagentaBright: [105, defaultBG],
   bgCyanBright: [106, defaultBG],
   bgWhiteBright: [107, defaultBG],
-};
+} as any as Record<string, [number, number]>;
```

Comment:
you should assign this object to a value and set the type there, rather than here. This force-casting is not a recommended approach which eliminates type script checker capabilities.

Reply:
It's the `__proto__: null` at the top that breaks this. I've replaced it with a ts-error instead.